include/odp/api/spec/packet.h

```diff
@@ -2035,6 +2035,110 @@ void odp_packet_shaper_len_adjust_set(odp_packet_t pkt, int8_t adj);
  */
 uint64_t odp_packet_cls_mark(odp_packet_t pkt);
 
+/**
+ * LSO options
+ */
+typedef struct odp_packet_lso_opt_t {
+	/** LSO profile handle
+	 *
+	 * The selected LSO profile specifies details of the segmentation operation to be done.
+	 * Depending on LSO profile options, additional metadata (e.g. L3/L4 protocol header
+	 * offsets) may need to be set on the packet. See LSO documentation
+	 * (e.g. odp_pktout_send_lso() and odp_lso_protocol_t) for additional metadata
+	 * requirements.
+	 */
+	odp_lso_profile_t lso_profile;
```

Comment:
It would be a loop as packet_io.h includes already odp/api/packet.h. Packet.h has references to e.g. odp_pktio_t already (build does not need that include).